src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -49,6 +49,8 @@ def _get_default_install_location(exe_name):
 name_arg_type = CliArgumentType(options_list=('--name', '-n'), metavar='NAME')
 
 register_cli_argument('acs', 'name', arg_type=name_arg_type, help='ACS cluster name', completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'))
+register_cli_argument('acs', 'resource_group', arg_type=resource_group_name_type, help='Name of the Azure resource group')
```

Comment:
You shouldn't need to override the help here as it's defined by `resource_group_name_type` for consistency throughout the CLI.
By defining help here, this help for `--resource-group` will have a different description than everywhere else.